tool/tsh/recording_export.go

```diff
@@ -104,7 +130,23 @@ loop:
 			switch evt := evt.(type) {
 			case *apievents.SessionStart:
 				return frameCount, trace.BadParameter("only desktop recordings can be exported")
+			case *apievents.WindowsDesktopSessionStart:
+				log.Debug("beginnning desktop recording export")
 			case *apievents.WindowsDesktopSessionEnd:
+				// We've reached the last event in the session, but there
+				// still be more time to play in the recording. This can happen
+				// for example if the user sits on an unchanging screen for a time
```

Comment:
Yeah granted it's a lot of hand wringing for dead air time.

Removed it in 6deb98e9d2fe926b4fb51b3f1dfaf8da767eb478, so now what this is changing is

1. Also removing dead air time from the beginning of the recording with `if previousFrameDelayMs == -1`
2. Moving frame logic out into the *apievents.DesktopRecording block, so that it's active even for non PNG events (like if the user is moving the mouse around and/or clicking, but that's not changing the screen). This is the change that made my recordings go from seeming crazy to being roughly in line with what I experienced during the session.
3. Accounting for `*apievents.WindowsDesktopSessionStart`